packages/react-lowcode/tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
The CLI will not work probably. We will probably need rollup to have two different targets.